packages/react/src/utils/assert.spec.ts

```diff
@@ -20,13 +21,14 @@ function get(type: 'Helper' | 'Core') {
 
 describe('assert', () => {
   it('should assert condition', () => {
```

Comment:
I think this is better to scope down it.

```suggestion
describe('assert', () => {
  // eslint-disable-next-line vitest/expect-expect
  it('should assert condition', () => {
```